### DISCUSSION TITLE
connecting local ravendb to server

### DISCUSSION THREAD

**Discussion by tobbin2** [ravendb/ravendb#15841] [General]
Hello,
we are currently hosting RavenDB on a kubernetes environment. Is there anyway to connect to that raven db instance instead of creating a local container for each developer. Getting really hard to configure each container each time.

**Comment by ayende** [comment]
Not sure what you mean.
You want to have a shared RavenDB server, which will be used by many people?
Or having a shared server that multiple independent local databases will pull data from?

If the later, did you see hub/sink replication? 
https://ravendb.net/docs/article-page/5.4/csharp/server/ongoing-tasks/hub-sink-replication#hubsink-replication
